docs/reference/snapshot-restore/repository-azure.asciidoc

```diff
@@ -182,7 +182,7 @@ is running.
 
 When running {es} in
 https://azure.microsoft.com/en-gb/products/kubernetes-service[Azure Kubernetes
-Service], for instance using {eck-ref}[{eck}], you should use
+Service], for instance using {eck-ref}/k8s-snapshots.html#k8s-azure-workload-identity[{eck}], you should use
```

Comment:
```suggestion
Service], for instance using https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-snapshots.html#k8s-basic-snapshot-azure[{eck}], you should use
```

Reply:
Hm that's not linking to the right subsection tho, https://github.com/elastic/cloud-on-k8s/pull/7980 removed `k8s-basic-snapshot-azure`.